src/external/roboto/roboto.css

```diff
@@ -0,0 +1,32 @@
+@font-face {
+    font-family: 'Roboto';
+    font-style: normal;
+    font-display: swap;
+    font-weight: 400;
+    src:
+        local('Roboto Regular '),
+        local('Roboto-Regular'),
+        url('./Roboto-Regular.ttf') format('truetype')
+}
+
+@font-face {
+    font-family: 'Roboto';
+    font-style: normal;
+    font-display: swap;
+    font-weight: 500;
+    src:
+        local('Roboto Medium '),
+        local('Roboto-Medium'),
+        url('./Roboto-Medium.ttf') format('truetype')
+}
+
+@font-face {
+    font-family: 'Roboto';
+    font-style: bold;
+    font-display: swap;
+    font-weight: 700;
+    src:
+        local('Roboto Bold '),
+        local('Roboto-Bold'),
+        url('./Roboto-Bold.ttf') format('truetype')
+}
```

Comment:
@minhducsun2002, thanks for importing Roboto. Make sure it comes with compiled build.

Reply:
```
$ tree build
build
├── asset-manifest.json
├── favicon.ico
├── index.html
├── manifest.json
├── precache-manifest.1fc2432fd96552e6ff36a42430dd3654.js
├── service-worker.js
└── static
    ├── css
    │   ├── main.d3bf29e6.chunk.css
    │   └── main.d3bf29e6.chunk.css.map
    ├── js
    │   ├── 2.da2b522a.chunk.js
    │   ├── 2.da2b522a.chunk.js.map
    │   ├── main.e091f3a0.chunk.js
    │   ├── main.e091f3a0.chunk.js.map
    │   ├── runtime~main.9eb600ee.js
    │   └── runtime~main.9eb600ee.js.map
    └── media
        ├── Roboto-Bold.7c181887.ttf
        ├── Roboto-Medium.d52f011b.ttf
        └── Roboto-Regular.18d44f79.ttf

4 directories, 17 files
```

Confirmed.